### ISSUE TITLE
setting entropy to TD_loss summary vars

### ISSUE THREAD

**Issue by ahmad-hl** [godka/Pensieve-PPO#10]
In `train.py`, why did you set `TD_loss` variable to entropy in `summary_vars[0]`?
You already computed avg_entropy in summary_vars[2]!

```
 summary_str = sess.run(summary_ops, feed_dict={
                    summary_vars[0]: actor._entropy,
                    summary_vars[1]: avg_reward,
                    summary_vars[2]: avg_entropy
                })
                writer.add_summary(summary_str, epoch)
                writer.flush()
```
Could you please clarify?

**Comment by godka** [comment]
Hi,
we have changed the variable name 'td_loss' to 'entropy weight'.
See details in https://github.com/godka/Pensieve-PPO/commit/3eed104b2ba9ef528f1dea43be0e835cd21560d7

**Comment by ahmad-hl** [comment]
why entropy_weight ?
Why not "td_loss"?
I still don't understand why do you track entropy_weight and avv_entropy and don't track the loss between policy and value function?

**Comment by godka** [comment]
The reason why we didn't monitor 'td_loss' is that such signals don't actually measure the performance.
See more in [this page](https://spinningup.openai.com/en/latest/spinningup/rl_intro3.html).

```
2. It doesn’t measure performance. A loss function usually evaluates the performance metric that we care about. Here, we care about expected return, J(\pi_{\theta}), but our “loss” function does not approximate this at all, even in expectation. This “loss” function is only useful to us because, when evaluated at the current parameters, with data generated by the current parameters, it has the negative gradient of performance.

But after that first step of gradient descent, there is no more connection to performance. This means that minimizing this “loss” function, for a given batch of data, has no guarantee whatsoever of improving expected return. You can send this loss to -\infty and policy performance could crater; in fact, it usually will. Sometimes a deep RL researcher might describe this outcome as the policy “overfitting” to a batch of data. This is descriptive, but should not be taken literally because it does not refer to generalization error.

We raise this point because it is common for ML practitioners to interpret a loss function as a useful signal during training—”if the loss goes down, all is well.” In policy gradients, this intuition is wrong, and you should only care about average return. The loss function means nothing.
```

Still, if you have to integrate TD_loss into Tensorboard, you can just rewrite build_summaries and actor.train with **actor.val_loss**.

https://github.com/godka/Pensieve-PPO/blob/3eed104b2ba9ef528f1dea43be0e835cd21560d7/src/train.py#L182
